service/prometheus/scrapeconfig.go

```diff
@@ -0,0 +1,61 @@
+package prometheus
+
+import (
+	"fmt"
+
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/config"
+	"k8s.io/client-go/pkg/api/v1"
+
+	"github.com/giantswarm/prometheus-config-controller/service/key"
+)
+
+// GetTarget takes a Kubernetes Service, and returns a LabelSet,
+// suitable for use as a target.
+func GetTarget(service v1.Service) model.LabelSet {
+	target := model.LabelSet{
+		model.LabelName(
+			fmt.Sprintf("%s.%s", service.Name, service.Namespace),
+		): "",
+	}
+
+	return target
+}
+
+// GetScrapeConfigs takes a list of Kubernetes Services,
+// and returns a list of Prometheus ScrapeConfigs.
+func GetScrapeConfigs(services []v1.Service) ([]config.ScrapeConfig, error) {
+	filteredServices := FilterServices(services)
+	groupedServices := GroupServices(filteredServices)
+
+	scrapeConfigs := []config.ScrapeConfig{}
+	for groupName, services := range groupedServices {
+		targets := []model.LabelSet{}
+		for _, service := range services {
+			targets = append(targets, GetTarget(service))
+		}
+
+		scrapeConfig := config.ScrapeConfig{
+			JobName: groupName,
+			Scheme:  "https",
```

Comment:
A constant with godoc would be nice for this.

Reply:
Done